.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes #
+
+#### Checklist
+
+- [ ] I have read the Contribution & Best practices Guide and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added/updated necessary tests/documentation (if applicable)
```

Comment:
I think decoupling tests and documentation would be a good idea as those are two very important units for sustainable development.